.github/workflows/backport.yml

```diff
@@ -16,7 +16,7 @@ jobs:
             exit 1
           fi
 
-      - uses: actions/checkout@v4
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
```

Comment:
I wonder what rennovate will do with this? Will it follow the pattern and update to the latest commit sha? Will it be able to retain the comment tying the sha back to a concrete release?

Reply:
See https://github.com/open-telemetry/opentelemetry-java/pull/7069/files#r1945589666

Reply:
even without https://github.com/open-telemetry/opentelemetry-java/pull/7069/files#r1945589666, Renovate does the right thing and updates the comment, we've had that setup (pinning but without https://github.com/open-telemetry/opentelemetry-java/pull/7069/files#r1945589666) for a while now in the instrumentation